core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3666,8 +3668,8 @@ private void setAttributeSingleFile(RpcContext rpcContext, LockedInodePath inode
         try (CloseableResource<UnderFileSystem> ufsResource = resolution.acquireUfsResource()) {
           UnderFileSystem ufs = ufsResource.get();
           if (ufs.isObjectStorage()) {
-            LOG.debug("setOwner/setMode is not supported to object storage UFS via Alluxio. "
-                + "UFS: " + ufsUri + ". This has no effect on the underlying object.");
+            SAMPLING_LOG.debug("setOwner/setMode is not supported to object storage UFS "
+                + "via Alluxio. UFS: {}. This has no effect on the underlying object.", ufsUri);
```

Comment:
Should we still sample this since it is already at a debug level?

Reply:
I see. In different versions, this line specifically was at `warn` so it would make sense to sample. I guess the question is, do we want this to be at warn and sample, or at debug and not sample?

Reply:
I just moved this back to LOG, but it is still on debug.